tribe-ext-template-outliner.php

```diff
@@ -203,87 +206,56 @@ public function get_all_options() {
 			return $settings->get_all_options();
 		}
 
+		/**
+		 * Enqueue styles.
+		 *
+		 * @since 1.0.0
+		 *
+		 * @return void
+		 */
 		function tribe_ext_template_outliner_styles() {
-			?><style> 
-				#tribe-ext-template-debug-panel { 
-					background-color: white; 
-					border-radius: 3px;
-					border: 1px solid black;
-					color: black; 
-					display: none; 
-					font-size: 0.8rem;
-					opacity: 90%;
-					padding: .5em 2em;
-					position: fixed; 
-					width: 50%;
-					z-index: 99999; 
-				}
-
-
-				#tribe-ext-template-debug-panel code {
-					font-size: 0.8rem;
-				}
-
-                #tribe-ext-template-debug-panel h1 {
-                    font-size: 16px;
-                    text-align: center;
-                }
-				#tribe-ext-template-debug-panel ul {
-					list-style: none;
-				}
-
-				#tribe-ext-template-debug-panel li { 
-					display: flex; 
-					margin-bottom: .25em; 
-					white-space:nowrap;
-				}
-                #tribe-ext-template-debug-panel span {
-                    width: 16em;
-                }
-                #tribe-ext-template-debug-panel a:after {
-                    content: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAQElEQVR42qXKwQkAIAxDUUdxtO6/RBQkQZvSi8I/pL4BoGw/XPkh4XigPmsUgh0626AjRsgxHTkUThsG2T/sIlzdTsp52kSS1wAAAABJRU5ErkJggg==);
-                    margin: 0 3px 0 5px;
-                }
-				#tribe-ext-template-debug-panel input { 
-					width: 100%;
-				}
-
-				.tribe-ext-template-debug {
-					display: none;
-				}
+			wp_enqueue_style( 
```

Comment:
We should register an asset separately from enqueuing it... and ideally use `tribe_asset()` instead